server.go

```diff
@@ -502,39 +499,43 @@ func (s *Server) HandleMetricPacket(packet []byte) error {
 // HandleTracePacket accepts an incoming packet as bytes and sends it to the
 // appropriate worker.
 func (s *Server) HandleTracePacket(packet []byte) {
-	s.Statsd.Incr("packet.received_total", nil, .1)
+	s.Statsd.Incr("ssf.received_total", nil, .1)
```

Comment:
Unrelated, do you mind adding a `// TODO Move this to some place that doesn't generate a huge # of metrics?`

I want to get rid of this using an in memory counter, cuz on some boxes we're generating thousands of packets from this.